packages/calipso/calipso.4.1.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "Rewrites C programs to remove non-structured control-flow"
+description:
+  "Calipso analyzes programs in order to replace all nonstructured instructions (i.e., break, return, switch...) by branches and, then, remove all branches. See https://dblp.org/rec/journals/tsi/CasseFRS02 for more details"
+maintainer: ["Ivan Gotovchits <ivg@ieee.org>"]
+authors: ["Hugues Cassé <casse@irit.fr> et al"]
+license: "LGPL-2.0-only"
+tags: ["FrontC" "C" "analysis"]
+homepage: "https://github.com/BinaryAnalysisPlatform/FrontC"
+bug-reports: "https://github.com/BinaryAnalysisPlatform/FrontC/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "2.7"}
+  "FrontC" {>= "4.0.0"}
```

Comment:
I suppose this is intended but just in case (since you did something different for ctoxml), are you sure you don't want to sync up the version as well?

Reply:
Indeed, I want.